cmd/oras/root/pull.go

```diff
@@ -54,7 +54,7 @@ func pullCmd() *cobra.Command {
 		Short: "Pull files from remote registry",
 		Long: `Pull files from remote registry
 
-Example - Pull artifact files from a registry:
+Example - Pull artifact files from a registry or an OCI layout folder:
```

Comment:
This example is to pull artifact files from a registry. It does not reflect the OCI layout folder.

Reply:
yeah, accepted the wrong replacement, will correct it.